src/main/resources/static/css/general.css

```diff
@@ -144,6 +138,13 @@ li {
     color: white;
     text-transform: uppercase;
 }
+
 div.active a {
-  color: #6ed2f0;
-}
\ No newline at end of file
+    color: #6ed2f0;
+}
+
+table, th, td {
```

Comment:
Try not to style the tag elements instead create a class you can then set to those tags
Also: there are classes that can be used for settting borders: https://getbootstrap.com/docs/4.6/content/tables/#bordered-table

And not sure about 'width: 30%'. This will propably break some other table (now or in the future)

Reply:
And why for table, th and td all alike?

Reply:
idk I haven't styled it